create/faq.md

```diff
@@ -180,20 +180,43 @@ Until our first stable release (v1.0), MeiliSearch minor versions are not compat
 
 ## What are the recommended requirements for hosting a MeiliSearch instance?
 
-**Short answer**: It's impossible to say. We recommend [experimenting with RAM availability and database size](#how-does-hardware-affect-search-speed) to find the hardware + relevancy setup that works best for your dataset.
+### Short Answer
 
-**Long answer**: The disk size and RAM usage of your MeiliSearch database can vary widely. The following factors have a great effect on space and memory use:
+It's extremely difficult to accurately estimate the size and memory use of a MeiliSearch database, even knowing the size of the `.JSON` files. There's just too much complexity involved in the indexing process. For a benchmark and more thorough explanation, [go here](/reference/under_the_hood/storage.md#measured-disk-usage).
+
+We recommend reading [the long answer below](#long-answer) to find the setup that works best for your needs.
+
+### Long Answer
+
+#### Memory Use
+
+When it comes to RAM, there are two things that can cause your usage to spike:
+
+1. Adding documents
+2. Updating index settings (if index contains documents)
+
+To reduce RAM usage, follow this best practice: always update index settings **before** adding your documents.
```

Comment:
What happen if you don't? Would be nice to give more explanation on why they should do that.